manifests/install.pp

```diff
@@ -30,11 +30,26 @@
     extract_path => $aerospike::download_dir,
     creates      => $dest,
     cleanup      => $aerospike::remove_archive,
-  } ~>
-  exec { 'aerospike-install-server':
-    command     => "${dest}/asinstall",
-    cwd         => $dest,
-    refreshonly => true,
+    notify       => Exec['aerospike-install-server'],
+  }
+
+  case $::osfamily {
+    'Debian': {
+      exec { 'aerospike-install-server':
+        command     => "${dest}/asinstall --force-confold -i",
```

Comment:
wouldn't it be cleaner to put that in a variable like:
```
$asinstall_params = $::osfamily ? {
'Debian' => " --force-confold -i",
default => "",
}
```

And then just change the command attribute to:
```
command     => "${dest}/asinstall ${asinstall_params}",
```

What do you think?